### PR TITLE
Handbook: Add structure to performance reviews

### PR DESCRIPTION
**Pull request by joepavitt**
## Description

Ahead of performance review season, I wanted to offer more structure to the reviews than we have seen in the past couple of years. This should offer more clarity to employees, and encourage an open conversation between managers and their team.

### Template:

Example Google Doc (linked in Handbook):

Screenshot 2024-08-30 at 11 47 53

### CODE REVIEW

**Review by ZJvandeWeg — APPROVED** [review]

File: src/handbook/peopleops/performance-review.md [hunk 1]
```diff
@@ -13,4 +13,43 @@ employee can still grow. This text is shared with each individual employee.
 While this performance review is written down, each manager should've communicated
 feedback regularly to minimize the chance for surprises during the written review.
 
-Performance review might influence the [compensation adjustment](../peopleops/compensation#adjustment).
+## Structure
+
+You can find a template document in Google Drive [here](https://docs.google.com/document/d/1E_Ni5kIcgRAvx9ZxFECOxJkFzzvDoUSpb5YfzKd1zCk/).
```
Comment (ZJvandeWeg) [inline src/handbook/peopleops/performance-review.md:18]: Nit: here as link text I think is a bit lazy

Comment (ZJvandeWeg) [reply src/handbook/peopleops/performance-review.md:18]:
```suggestion
A [template document](https://docs.google.com/document/d/1E_Ni5kIcgRAvx9ZxFECOxJkFzzvDoUSpb5YfzKd1zCk/) is available to structure the review.
```

File: src/handbook/peopleops/performance-review.md [hunk 2]
```diff
@@ -13,4 +13,43 @@ employee can still grow. This text is shared with each individual employee.
...
+if given "Room for Growth" feedback, in fact, managers are encouraged to assign this rating to at least one of the
+company values, although this should not be forced.
+
+It is almost impossible to achieve/exceed all values perfectly, sometimes particular employees are more focus on
+customer/community engagement, others may be more focussed on specific technical requirements. This is normal and should be embraced.
+
+### Highlights
+
+A chance to focus on the best things that the employee has contributed to FlowFuse since their last review (or first day
+if this is their first review). Highlights should come from both employees and their managers, and are a chance to reflect
```
Comment (ZJvandeWeg) [inline src/handbook/peopleops/performance-review.md:41]: > Highlights should come from both employees and their managers

How do others inform a manager of highlights of their peer? It seems like the manager should be observant?

Comment (joepavitt) [reply src/handbook/peopleops/performance-review.md:41]: Good point - I had thought about 360 Feedback too, but didn't want to introduce too much structure. We do also have the [Slack bots](https://flowfuse.com/handbook/company/communication/#feedback-and-thanks) for this, but I don't think I've ever seen anyone utilise them.

Comment (ZJvandeWeg) [reply src/handbook/peopleops/performance-review.md:41]: Point of feedback for the review! :)

**Pull request merged**